src/main/java/org/openstreetmap/atlas/generator/AtlasGenerator.java

```diff
@@ -270,15 +238,9 @@ public void start(final CommandMap command)
                                 AtlasGeneratorJobGroup.FULLY_SLICED.getCacheFolder()),
                         atlasScheme, tasks));
         countryAtlasShardsRDD.cache();
-
-        if (useJavaFormat)
-        {
-            saveAsHadoop(countryAtlasShardsRDD, AtlasGeneratorJobGroup.WAY_SECTIONED, output);
-        }
-        else
-        {
-            saveAsHadoop(countryAtlasShardsRDD, AtlasGeneratorJobGroup.WAY_SECTIONED_PBF, output);
```

Comment:
oh good-- can we remove the `AtlasGeneratorJobGroup.WAY_SECTIONED` enum value too?

Reply:
Oh yes, good catch